tm2/pkg/crypto/keys/client/maketx.go

```diff
@@ -219,6 +220,7 @@ func ExecSignAndBroadcast(
 	io.Println("GAS USED:  ", bres.DeliverTx.GasUsed)
 	io.Println("HEIGHT:    ", bres.Height)
 	io.Println("EVENTS:    ", string(bres.DeliverTx.EncodeEvents()))
+	io.Println("TX HASH:   ", hex.EncodeToString(bres.Hash))
```

Comment:
Same as the previous comment about base64 encoding 🙏

Reply:
https://github.com/gnolang/gno/pull/2309/commits/c1eb1fe7f6ce25dc3389b905e79b3799312e2c86